generate_bindings.sh

```diff
@@ -29,6 +31,8 @@ echo ""
 echo "Generating bindings for $PREFERENCES_HEADER_PATH"
 bindgen \
     --no-doc-comments \
+    --use-core \
+    --ctypes-prefix "::libc" \
```

Comment:
I think the `::` can be left out. Since a few versions back Rust automatically grabs the global namespace without the preceding double colon. But that's a nitpick.